firefox-ios/Client/Frontend/Settings/Main/Support/StudiesToggleSetting.swift

```diff
@@ -39,9 +40,36 @@ class StudiesToggleSetting: BoolSetting {
                 Experiments.setStudiesSetting($0)
             }
         )
-        // We make sure to set this on initialization, in case the setting is turned off
-        // in which case, we would to make sure that users are opted out of experiments
-        Experiments.setStudiesSetting(prefs.boolForKey(AppConstants.prefStudiesToggle) ?? true)
+
+        setupSettingDidChange()
+
+        let sendUsageDataPref = prefs.boolForKey(AppConstants.prefSendUsageData) ?? true
+
+        // Special Case (EXP-4780) disable studies if usage data is disabled
+        updateSetting(for: sendUsageDataPref)
+    }
+
+    private func setupSettingDidChange() {
+        self.settingDidChange = {
+            Experiments.setStudiesSetting($0)
+        }
+    }
+
+    func updateSetting(for isUsageEnabled: Bool) {
+        guard !isUsageEnabled else {
+            // Note: switch should be enabled only when telemetry usage is enabled
+            control.setSwitchTappable(to: true)
+            // We make sure to set this on initialization, in case the setting is turned off
+            // in which case, we would to make sure that users are opted out of experiments
+            Experiments.setStudiesSetting(prefs?.boolForKey(AppConstants.prefStudiesToggle) ?? true)
+            return
+        }
+
+        // Special Case (EXP-4780) disable Studies if usage data is disabled
+        control.setSwitchTappable(to: false)
+        control.toggleSwitch(to: false)
+        writeBool(control.switchView)
```

Comment:
`writeBool(control.switchView)` is technically writing when usage is false. We don't do it for `true` because we aren't making the switch go back to true. 

When we the toggleSwitch to false this ensures that writing false on the next launch of settings keeps it that way because prefs `prefStudiesToggle` gets written to disk by BoolSettings class.

Reply:
I can walk you through more, maybe I missed something but this is me being overcautious and we rather have it false than put a user into a study they didn't enrol in :)

Reply:
```
Maybe we need to keep the user's last preference for the study toggle?

If a user originally had studies ON, then disabled telemetry, then re-enabled telemetry... what do we expect should happen with studies? ON, or OFF for that user?
```

I had the same thought initially and was going to persist but I went with the opposite approach that its safe to keep it OFF rather than automatically toggling back to the original state. They could very well be be in a study that is no longer applicable if they go back to the settings. So my thinking is this is done to ensure we show the user our intent on how it works but it still requires copy change in future to inform them. Lots to think here so went with the safe bet of turning it OFF. We can easily do the opposite if PMs decide otherwise and should revisit in your Settings IA for more clear option. [ref](https://github.com/mozilla-mobile/firefox-ios/pull/21917#discussion_r1757853666)